### ISSUE TITLE
Can't read SOLPS shots from MDSplus with MDS+ 7.49

### ISSUE THREAD

**Issue by jacklovell** [cherab/solps#2]
### Steps to reproduce:
1. Install MDSplus 7 (e.g. from the stable repository via mdsplus.org)
1. Run the following code:
```python
from cherab.solps import load_solps_from_mdsplus                                                 
sim = load_solps_from_mdsplus('solps-mdsplus.aug.ipp.mpg.de:8001', 69636)                        
```

### Expected result
Loads successfully, just like it does with MDSplus 6.1 (on Freia)

### Actual result
```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-335c3f87ca7d> in <module>
----> 1 sim = load_solps_from_mdsplus('solps-mdsplus.aug.ipp.mpg.de:8001', 69636)

~/Software/anaconda3/lib/python3.7/site-packages/cherab/solps/formats/mdsplus.py in load_solps_from_mdsplus(mds_server, ref_number)
     83 
     84     # Master list of species, e.g. ['D0', 'D+1', 'C0', 'C+1', ...
---> 85     sim._species_list = conn.get('\SOLPS::TOP.IDENT.SPECIES').data().decode('UTF-8').split()
     86     sim._species_density = np.swapaxes(conn.get('\SOLPS::TOP.SNAPSHOT.NA').data(), 0, 2)
     87     sim._rad_par_flux = np.swapaxes(conn.get('\SOLPS::TOP.SNAPSHOT.FNAY').data(), 0, 2)  # radial particle flux

AttributeError: 'numpy.str_' object has no attribute 'decode'
```

### Explanation
It seems like between MDSplus 6 and 7 the return type of string variables changed from `numpy.bytes_` to `numpy.str_`. The latter does not require decoding to a UTF-8 string. This could be handled with a relatively simple try/except block.

**Comment by mattngc** [comment]
@jacklovell If you've already got any easy fix for this in your own code, please do a PR for it.

**Comment by jacklovell** [comment]
Fixed by #4
